app/models/group.rb

```diff
@@ -483,13 +483,21 @@ def self.refresh_automatic_group!(name)
         "SELECT id FROM users WHERE id <= 0 OR trust_level < #{id - 10} OR staged"
       end
 
-    DB.exec <<-SQL
+    removed_user_ids = DB.query_single <<-SQL
       DELETE FROM group_users
             USING (#{remove_subquery}) X
             WHERE group_id = #{group.id}
               AND user_id = X.id
+      RETURNING group_users.user_id
     SQL
 
+    if removed_user_ids.present?
+      Jobs.enqueue(
+        :publish_group_membership_updates,
+        user_ids: removed_user_ids, group_id: group.id, type: :remove
```

Comment:
@romanrizzi I believe the using symbols in the `type:` arg is responsible for these deprecation warnings:

```
Deprecation notice: Jobs::PublishGroupMembershipUpdates was enqueued with argument values which do not cleanly serialize to/from JSON. This means that the job will be run with slightly different values than the ones supplied to `enqueue`. Argument values should be strings, booleans, numbers, or nil (or arrays/hashes of those value types). (deprecated since Discourse 2.9) (removal in Discourse 3.0)
At app/models/group.rb:494:in `refresh_automatic_group!`
```

Reply:
Thanks for letting me know! Will open a PR to fix the deprecation.